tamr_toolbox/data_io/dataframe.py

```diff
@@ -338,6 +365,7 @@ def validate(
         require_unique_columns: list of columns that are checked to have all unique values,
             e.g. a primary key column
         require_nonnull_columns: list of columns that are checked to have all non-null values
+        custom_check_columns: list containing custom function and list of columns to apply function too
```

Comment:
Slight wording change to suggest the "tuple"
```suggestion
        custom_check_columns: tuple of custom function and list of columns to apply function to
```